### PR TITLE
Util function to read BT rows in batch and in parallel, and use that in GetPropertyValues/Labels.

### PR DESCRIPTION
**Pull request by Spaceenter**
Will use the same util function for the rest of the API methods wherever appropriate.

### CODE REVIEW

File: store/arc.go [hunk 1]
```diff
@@ -301,73 +300,47 @@ func (s *store) btGetPropertyValues(ctx context.Context,
...
-			nodeList := make([]Node, 0)
-			for _, btRawValue := range btRawValues {
-				btJSONRaw, err := util.UnzipAndDecode(string(btRawValue))
-				if err != nil {
-					return nil, err
-				}
+	for dcid, btRawValues := range btRawValuesMap {
+		nodeList := make([]Node, 0)
+		for _, btRawValue := range btRawValues {
+			btJSONRaw, err := util.UnzipAndDecode(string(btRawValue))
```
Comment (shifucun) [inline store/arc.go]: Maybe should start test if we just save the original data and save this step.

File: util/util.go [hunk 2]
```diff
@@ -237,3 +240,70 @@ func SnakeToCamel(s string) string {
...
+	case bigtable.RowRangeList:
+		rowRangeList = rowSet.(bigtable.RowRangeList)
+		rowSetSize = len(rowRangeList)
+	default:
+		return fmt.Errorf("unsupported RowSet type: %v", v)
+	}
+
+	errs, errCtx := errgroup.WithContext(ctx)
+	rowChan := make(chan []bigtable.Row, rowSetSize)
+	for i := 0; i <= rowSetSize/1000; i++ {
```
Comment (shifucun) [inline util/util.go]: make this a constant?

File: go.sum [hunk 3]
```diff
@@ -29,6 +29,7 @@ github.com/google/datacommons v0.0.0-20190708184655-a56d2ce8bba9 h1:FSyDFThfX8Ts
 github.com/google/go-cmp v0.2.0/go.mod h1:oXzfMopK8JAjlY9xF4vHSVASa0yLyX7SntLO5aqRK0M=
 github.com/google/go-cmp v0.3.0 h1:crn/baboCvb5fXaQ0IJ1SGTsTVrWpDsCWC8EGETZijY=
 github.com/google/go-cmp v0.3.0/go.mod h1:8QqcDgzrUqlUb/G2PQTWiueGozuR1884gddMywk6iLU=
+github.com/google/go-cmp v0.3.1 h1:Xye71clBPdm5HgqGwUkwhbynsUJZhDbS20FvLhQ2izg=
```
[the remark this answers is not in the document]
Comment (Spaceenter) [reply go.sum]: https://www.reddit.com/r/golang/comments/97dxj1/should_i_add_the_gosum_file_to_the_repo_of_a/

File: store/arc.go [hunk 4]
```diff
@@ -283,7 +283,6 @@ func (s *store) btGetPropertyValues(ctx context.Context,
 		direction = "in"
 	}
 	valType := in.GetValueType()
-	btTable := s.btClient.Open(util.BtTable)
 
 	keyPrefix := func(out bool) string {
```
[the remark this answers is not in the document]
Comment (Spaceenter) [reply store/arc.go]: Done.

File: store/arc.go (hunk 1 above)
Comment (Spaceenter) [reply store/arc.go]: Yes, for GetPropertyValues, we do plan to remove compression and b64 encoding. Will change this when BT data is updated.

File: util/util.go (hunk 2 above)
Comment (Spaceenter) [reply util/util.go]: Done.

**Pull request merged**